Monika After Story/game/script-topics.rpy

```diff
@@ -4670,7 +4670,8 @@ init 5 python:
     addEvent(Event(persistent.event_database,eventlabel="monika_ribbon",category=['monika'],prompt="Ribbons",random=True))
 
 label monika_ribbon:
-    if not monika_chr.is_wearing_acs_types("ribbon", "twin-ribbons", "s-type-ribbon"):
+    # TODO: We need a better handling for this
+    if not monika_chr.is_wearing_acs_types("ribbon", "twin-ribbons", "s-type-ribbon", "bow"):
```

Comment:
The retro bows shouldn't be "bow" type. They look like a standard ribbon so they should have been the ribbon acs-type.

Reply:
Not sure why we set it to `bow`, but changing that now will mean people will have to update their spritepacks to apply the fix.

Reply:
we can add bow here if multi's black bow should count as a ribbon, but otherwise we need to change the retro ribbons spritepacks regardless. the type and naming is wrong.

Reply:
@multimokia decide.

But I'd change the spritepack in a new PR.